src/pages/index.js

```diff
@@ -83,12 +104,11 @@ function getCardElement(data) {
     previewModalImageEl.alt = cardNameEl.textContent;
   });
 
```

Comment:
* When I reload the page, all **red** hearts become **transparent** again. You need to check `isLiked` from the card data





```js


  if (data.isLiked) {
    cardLikeBtn.classList.add("card__like-button-liked");
  }
```